scripts/jenkins-yoctobuild-build.sh

```diff
@@ -220,20 +239,49 @@ fi
 
 
 if [ "$RUN_INTEGRATION_TESTS" = "true" ]; then
-    cd $WORKSPACE
-    # Set build dir for qemu again, BBB build might possibly have overridden
-    # this.
-    source oe-init-build-env build-qemu
-    prepare_and_set_PATH
+      # this isn't a PR test
+      if [ -z $PR_TO_TEST ]; then
 
+      cd $WORKSPACE
+      # Set build dir for qemu again, BBB build might possibly have overridden
+      # this.
+      source oe-init-build-env build-qemu
+      prepare_and_set_PATH
 
-    cd $WORKSPACE/meta-mender/meta-mender-qemu
-    cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/core-image-full-cmdline-vexpress-qemu.{ext4,sdimg} .
-    cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/u-boot.elf .
 
-    docker build -t mendersoftware/mender-client-qemu:pr --build-arg VEXPRESS_IMAGE=core-image-full-cmdline-vexpress-qemu.sdimg --build-arg UBOOT_ELF=u-boot.elf .
-    $WORKSPACE/integration/extra/release_tool.py --set-version-of mender --version pr
-    cd $WORKSPACE/integration/tests && ./run.sh
+      cd $WORKSPACE/meta-mender/meta-mender-qemu
+      cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/core-image-full-cmdline-vexpress-qemu.{ext4,sdimg} .
+      cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/u-boot.elf .
+
+
+      # only create a mender pr container if we build qemu
+      if [ "$BUILD_QEMU" = "true" ]; then
```

Comment:
essentially you want me to drop the test and just leave the container building inside it's current block, right?